packages/react-core/package.json

```diff
@@ -46,8 +46,11 @@
     "react-hook-form": "^7.43.5",
     "xstate": "^4.33.6"
   },
+  "devDependencies": {
+    "@aws-amplify/core": "^6.4.0"
+  },
```

Comment:
Let's list in the top level devDeps with the rest of the `aws-amplify` related scoped packages

Reply:
Moved